sklearn/preprocessing/data.py

```diff
@@ -97,9 +97,11 @@ def scale(X, axis=0, with_mean=True, with_std=True, copy=True):
     with_mean : boolean, True by default
         If True, center the data before scaling.
 
-    with_std : boolean, True by default
-        If True, scale the data to unit variance (or equivalently,
-        unit standard deviation).
+    with_std : boolean, 1 or 2, True by default
```

Comment:
Isn't this confusing a bit? As a user, I'd expect to be able to give any floating point value as the input, if the API's accepting numbers for the scaling factor. Maybe string literals `'one', 'two'` would be more appropriate? Otherwise, I don't see why we'd limit the user to these two values.

Reply:
Right we shouldn't limit user to these two values. Instead it should be `'int'`

Reply:
or `float`

Reply:
but I'd like to know what @jnothman thinks about this one.